src/sentry/seer/anomaly_detection/store_data.py

```diff
@@ -115,7 +132,19 @@ def send_historical_data_to_seer(alert_rule: AlertRule, project: Project) -> Bas
         )
         raise TimeoutError
 
-    # TODO warn if there isn't at least 7 days of data
+    MIN_DAYS = 7
+    data_start_index, data_end_index = _get_start_and_end(historical_data)
+    if data_start_index == -1:
+        resp.status = 202
+        resp.reason = NOT_ENOUGH_DATA  # this reason value isn't read anywhere, but I think it's good to justify the 202
+
+    data_start_time = datetime.fromtimestamp(
+        historical_data.data.get("data")[data_start_index]["time"]
```

Comment:
not a huge deal but since we're checking after the data has been formatted you could use `formatted_data` instead and do `formatted_data[data_start_index].get("timestamp")`